Tests/Tests.csproj

```diff
@@ -13,6 +13,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="3.8.0" />
```

Comment:
Why is this reference necessary?

Reply:
hmm... it's not necessary, I will delete it.